run.sh

```diff
@@ -344,6 +360,10 @@ parse_options() {
       fi
       shift 2
       ;;
+    -u | --update-image)
```

Comment:
`--update-image` sounds like it triggers an in-image update. Maybe something like `--refresh-cache` or `--overwrite-cache`?

Now that I think about this, maybe trunk/alpha images should never be cached, or always overwritten. For dev purposes, we could also provide the inverse option, something like `--force-cache`.

BTW: we also already have `--clean` for cleaning the cache.

Reply:
Yes, those sound better.

> Now that I think about this, maybe trunk/alpha images should never be cached, or always overwritten. For dev purposes, we could also provide the inverse option, something like `--force-cache`.

That was the main intent of this option. `--clean` clears the entire cache, I simply wanted to use the latest image for trunk / alpha.